elastic/kibana
# chore(slo): remove space slo resources

## 📝 Summary

Resolves https://github.com/elastic/kibana/issues/141527

This PR removes the space aware destination indices to only keep a global index shared across all spaces, thus keeping to a minimum the heap usage of the elasticsearch nodes.

## ✅ Manual Testing

1. Create an SLO from default space
2. Create another SLO from "other" space
3. Check transforms are both running, and writing to the same index `slo-observability.sli-v1.YYYY-MM-DD` using the same ingest pipeline
4. Delete the SLO
5. Check the related transforms and data is removed from the destination index
